rskj-core/src/test/java/org/ethereum/rpc/Web3ImplLogsTest.java

```diff
@@ -51,6 +51,39 @@
  * Created by ajlopez on 30/11/2016.
  */
 public class Web3ImplLogsTest {
+
+
+    // Events used:
+    // event Incremented(bool indexed odd, uint x);
+    // event Created(uint x);
+    // event Valued(uint x);
+
+
+    /////////////////////////////////////////////////////////////////////////
+    // IMPORTANT INFORMATION WHEN WORKING WITH SOLIDITY GENERATED TOPICS
+    // FROM EVENTS
+    // 1. The event MUST be converted to its normalized form first.
+    // 2. uint is not a normalized type. uint -> uint256
+    // Note the part that "In Solidity: The first topic is the hash of the signature of the event."
+    // Canonical types, such as uint256 have to be used in signatures.
+    // 3. The signature is built by removing all argument names (only types are left)
+    // 4. "indexed"  word must not be present
+    // 5. Case is important: do not change upper/lower case
+    // 6. The topic is the Keccak-256 hash digest of the signature.
+    // 6. web3.sha3() IS NOT SHA3! It's Keccak-256. Solidity signatures use Keccak-256. NOT SHA3.
+    //
+    // Examples:
+    // Incremented(bool indexed odd, uint x) -> Keccak-256("Incremented(bool,uint256)")
+    //
+    String GetValuedEventSignature="1ee041944547858a75ebef916083b6d4f5ae04bea9cd809334469dd07dbf441b";
+
+
+    String IncrementMethodSignature ="371303c0";
+    String GetValueMethodSignature ="20965255";
```

Comment:
This goes against our code conventions. What about converting them to constants like
```java
private final static String GET_VALUED_EVENT_SIGNATURE = "1ee041944547858a75ebef916083b6d4f5ae04bea9cd809334469dd07dbf441b";
private final static String INCREMENT_METHOD_SIGNATURE = "371303c0";
private final static String GET_VALUE_METHOD_SIGNATURE = "20965255";
```

Reply:
Same review.